stps/sig-network/ip-request-stp.md

```diff
@@ -0,0 +1,218 @@
+
+**Document Conventions (if applicable):** None
+
+#### **2. Testing Goals**
+
+Define specific, measurable testing objectives for this feature, such as:
+
+- [ ] Goal 1: Verify known networking scenarios are still valid when primary static IP is assigned via the new\
+annotation.
+- [ ] [Goal 2: Consider new cases to be tested, presented by this new feature.
+
+#### **3. Non-Goals (Testing Scope Exclusions)**
+
+Explicitly document what is **out of scope** for testing. **Critical:** All non-goals require explicit stakeholder agreement to prevent "I assumed you were testing that" issues.
+
+**Note:** Replace example rows with your actual non-goals. Each non-goal must have PM/Lead sign-off.
+
+| Non-Goal                                            | Rationale                                                                      | PM/ Lead Agreement |
+|:----------------------------------------------------|:-------------------------------------------------------------------------------|:-------------------|
+| Templating VM spec with the `addresses` annottaion. | It's the responsibility of the 3rd party integrator, and not accessible by us. | [ ] Name/Date      |
+| Creating C/UDN with IP range.                       | Same                                                                           | [ ] Name/Date      |
+
+
+**Important Notes:**
+- Non-goals without stakeholder agreement are considered **risks** and must be escalated (see Section II.7 - Risks and Limitations)
+- Review non-goals during Developer Handoff/QE Kickoff meeting (see Section I.2 - Technology and Design Review)
+
+#### **4. Test Strategy**
+
+##### **A. General**
+Testing this feature should consider and cover the following:
+* see the IP address in the VMI status
+* see it in the guest
+* verify the address is pingable
+* verify this interface can maintain TCP connectivity
+* the feature should be applied on both IPv4/6 families
+* verify the assigned address and connectivity are maintained after VM reboot
+* verify the assigned address and connectivity are maintained after VM migration
+* verify the assigned address can be reclaimed and re-used after deleting the VM
+
+
+##### **B. Types of Testing**
+
+The following types of testing must be reviewed and addressed.
+
+**Note:** Mark "Y" if applicable, "N/A" if not applicable (with justification in Comments). Empty cells indicate incomplete review.
+
+| Item (Testing Type)            | Applicable (Y/N or N/A) | Comments |
+|:-------------------------------|:------------------------|:---------|
+| Functional Testing             | Y                       |          |
+| Automation Testing             | Y                       |          |
+| Performance Testing            | N/A                     |          |
+| Security Testing               | N/A                     |          |
+| Usability Testing              | Y                       |Manual testing of the feature in UI, no automation.          |
```

Comment:
based on https://access.redhat.com/solutions/7133388is there a plan to add this explicitly in the ui? 
https://access.redhat.com/solutions/7133388 i guess the answer is `no`